### PR TITLE
Updated JES spec to warn and not crash on short AES keys.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

File: supportedBackends/jes/src/test/scala/cromwell/backend/impl/jes/JesInitializationActorSpec.scala [hunk 1]
```diff
@@ -132,7 +132,7 @@ class JesInitializationActorSpec extends TestKitSuite("JesInitializationActorSpe
       |}
       | """.stripMargin))
 
-  val defaultBackendConfig = new BackendConfigurationDescriptor(backendConfig, globalConfig)
+  val defaultBackendConfig = BackendConfigurationDescriptor(backendConfig, globalConfig)
```
Comment (cjllanwarne) [inline supportedBackends/jes/src/test/scala/cromwell/backend/impl/jes/JesInitializationActorSpec.scala]: While you're here, might as well go ahead and make `BackendConfigurationDescriptor` a `final case class`

**Comment by cjllanwarne** [comment]
👍

File: supportedBackends/jes/src/test/scala/cromwell/backend/impl/jes/JesInitializationActorSpec.scala (hunk 1 above)
Comment (kshakir) [inline supportedBackends/jes/src/test/scala/cromwell/backend/impl/jes/JesInitializationActorSpec.scala]: Another PR. I was just fixing the yellow in this file. ;)

**Comment by Horneth** [comment]
👍

**Pull request merged**